src/back/hlsl/storage.rs

```diff
@@ -1,7 +1,6 @@
 //! Logic related to `ByteAddressBuffer` operations.
 //!
 //! HLSL backend uses byte address buffers for all storage buffers in IR.
-//! Matrices have to be transposed, because HLSL syntax implies row majority.
```

Comment:
can we have 1-2 sentence description of this matrix logic in `src/back/hlsl/mod.rs` header?